magenta-lib/src/main/scala/magenta/input/resolver/TaskResolver.scala

```diff
@@ -8,15 +9,15 @@ import magenta.{App, DeployParameters, DeployTarget, DeploymentPackage, Deployme
 
 object TaskResolver {
   def resolve(deployment: Deployment, deploymentResources: DeploymentResources, parameters: DeployParameters,
-    deploymentTypes: Seq[DeploymentType], artifact: S3Artifact): Either[ConfigError, DeploymentTasks] = {
+    deploymentTypes: Seq[DeploymentType], artifact: S3Artifact): ValidatedNel[ConfigError, DeploymentTasks] = {
     val deploymentPackage = createDeploymentPackage(deployment, artifact)
-    val deploymentTypeEither = deploymentTypes.find(_.name == deployment.`type`).
-      toRight(ConfigError(deployment.name, s"Deployment type ${deployment.`type`} not found"))
+    val validatedDeploymentType = Validated.fromOption(deploymentTypes.find(_.name == deployment.`type`),
+      NEL.of(ConfigError(deployment.name, s"Deployment type ${deployment.`type`} not found")))
```

Comment:
Yep - that was before I started aliasing `NonEmptyList => NEL` in all the imports. Am thinking about a global type alias.